common/country_types/hfe_country_types.txt

```diff
@@ -231,13 +283,12 @@ hfe_drone = {
 		category = country_base
 		produces = {
 			influence = 3
+			minerals = 50
+			alloys = 50
+			energy = 50
 		}
 		produces = {
-			trigger = {
-				is_difficulty > 0
-			}
-			physics_research = 10
-			society_research = 10
+			physics_research = 1
```

Comment:
This will make drones too weak

Reply:
They'll get their Physics from mining stations and they have either Tier Four or Tier Five technology, one or two steps below a Vanilla rendition Fallen Empire. They also have a fleet over ten thousand strong. They won't be in trouble.